library/src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -21,6 +21,7 @@
 
 import com.pokegoapi.api.device.DeviceInfo;
 import com.pokegoapi.api.device.DeviceInfos;
+
```

Comment:
can you try to remove all the changes from this file, because it doesn´t change to keep the filehistory clean.